aten/src/ATen/cuda/detail/CUDAHooks.h

```diff
@@ -49,6 +49,8 @@ struct CUDAHooks : public at::CUDAHooksInterface {
   bool hasCUDART() const override;
   long versionCUDART() const override;
   long versionCuDNN() const override;
+  long versionRuntimeCuDNN() const override;
+  long versionCuDNNFrontend() const override;
```

Comment:
Why does Runtime CUDNN frontend matter? It cannot be changed right? It's a compile time include header?

Reply:
I sidecar'd this change in as we'll need it in the near future for SDPA issues that require a cuDNN frontend version to be available for gating. In theory `sdp_utils.cpp` could be able to access this but I'm not sure I want to include that directly.

Reply:
Can the runtime version be different for cudNNFronteEnd or should it be constexpr?